README.md

```diff
@@ -25,7 +25,7 @@ The first time you run `git commit` **in a repository** will throw a warning to
 ```shell
 Unable to open baseline file: REPO_ROOT/.secrets.baseline
 Please create it via
-   detect-secrets scan > $HOME/<your-git-repo>/.secrets.baseline
+   detect-secrets scan > /<path-to-your-git-repo>/.secrets.baseline
```

Comment:
I set this to an insert path here kind of thing because Ollie changed it so you could install it anywhere.

Reply:
🥇